lexicons/tools/ozone/auth/reporter.json

```diff
@@ -0,0 +1,13 @@
+{
+  "lexicon": 1,
+  "id": "tools.ozone.auth.reporter",
+  "defs": {
+    "main": {
+      "type": "auth-bundle",
+      "description": "Report content and view status of reports",
+      "permissions": [
+        "com.atproto.resource.authentication?aud=*&lxm=tools.ozone.moderation.createReport"
```

Comment:
implies moving this method into the ozone namespace which I think we're down with (can keep around the com.atproto method for now as well)